apps/challenges/models.py

```diff
@@ -260,6 +260,39 @@ def is_active(self):
         return False
 
 
+def slack_challenge_approval_callback(challenge_id):
```

Comment:
A function similar to `create_eks_cluster_or_ec2_for_challenge` but used for Slack (cause we don't have `instance` and `created` if we call this function in our backend)